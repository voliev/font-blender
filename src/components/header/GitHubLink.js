import React from "react";
import GitHub from "@material-ui/icons/GitHub";
import "./styles/GitHubLink.css";

export const GitHubLink = () => (
  <a
    aria-label="Project GitHub page"
    className="GitHubLink"
    href="https://github.com/voliev/font-blender"
    rel="noopener noreferrer"
    target="_blank"
  >
    <GitHub fontSize="large" />
  </a>
);

export default React.memo(GitHubLink);
