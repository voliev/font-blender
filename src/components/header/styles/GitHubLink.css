.GitHubLink {
  margin: 0 var(--space-lg) 0 0;
}

.GitHubLink {
  align-items: center;
  color: var(--white);
  display: flex;
  height: 2.5rem;
  justify-content: center;
  width: 2.5rem;
}
