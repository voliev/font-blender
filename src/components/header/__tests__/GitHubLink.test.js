import React from "react";
import { shallow } from "enzyme";
import { GitHubLink } from "../GitHubLink";

it("should render without crashing", () => {
  const wrapper = shallow(<GitHubLink />);

  expect(wrapper).toMatchSnapshot();
});
