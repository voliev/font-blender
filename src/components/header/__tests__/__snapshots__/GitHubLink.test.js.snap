// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crashing 1`] = `
<a
  aria-label="Project GitHub page"
  className="GitHubLink"
  href="https://github.com/voliev/font-blender"
  rel="noopener noreferrer"
  target="_blank"
>
  <GitHubIcon
    fontSize="large"
  />
</a>
`;
